scripts/jenkins/Jenkinsfile.monthly

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env groovy
+
+stage('Trigger monthly libelektra/master build') {
+    build job: 'libelektra/master', wait: false
```

Comment:
In the other Jenkinsfile we use two spaces.

Reply:
@Mistreated can you unify the style (use two spaces) and rebase so we can merge this?